powerauth-test-server/pom.xml

```diff
@@ -198,21 +182,24 @@
 
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
+                <artifactId>maven-resources-plugin</artifactId>
                 <executions>
                     <execution>
-                        <id>enforce-banned-dependencies</id>
+                        <id>copy-resources</id>
+                        <phase>process-resources</phase>
                         <goals>
-                            <goal>enforce</goal>
+                            <goal>copy-resources</goal>
                         </goals>
                         <configuration>
-                            <rules>
-                                <bannedDependencies>
-                                    <excludes>
-                                        <exclude>org.apache.tomcat.embed:*:*:*:compile</exclude>
-                                    </excludes>
-                                </bannedDependencies>
-                            </rules>
+                            <outputDirectory>${basedir}/target/classes/db/changelog</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>${project.parent.basedir}/docs/db/changelog/changesets/${project.artifactId}</directory>
+                                    <includes>
+                                        <include>**/*.*</include>
+                                    </includes>
+                                </resource>
+                            </resources>
```

Comment:
@romanstrobl @zcgandcomp Do we want to run liquibase changes automatically or manually on demand only?
@jnpsk In that case, just remove `liquibase-core` dependency and it turns off the spring boot autoconfiguration of liquibase.

Reply:
@banterCZ @jnpsk  On the topic -  https://github.com/wultra/powerauth-tests/pull/279

Reply:
Solved in https://github.com/wultra/powerauth-tests/pull/279